packages/jest-runtime/src/index.ts

```diff
@@ -294,6 +298,9 @@ export default class Runtime {
           const moduleID = this._resolver.getModuleID(
             this._virtualMocks,
             filePath,
+            undefined,
+            // is this correct?
```

Comment:
automock is weird, and this should be a fully resolved path already, so I guess we can remove it as it doesn't matter